README.md

```diff
@@ -43,3 +45,13 @@ Coverage analysis is based on ctest. To launch coverage, launch from source dire
 `ctest -S cmake/CTestScript.cmake -DDYNAWO_HOME=<DYNAWO_HOME> -DBOOST_ROOT=<DYNAWO_HOME> -VV`
 
 Outputs are gcov and lcov reports
+
+## Developement standard
+### Check style
+The check style, based on Dynawo's, is based on a [clang-format file](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) using [clang-format](https://clang.llvm.org/docs/ClangFormat.html). It is recommended for developpers to apply it before commiting to ensure coherence with future developpements.
+
+### SCM
+git commits have a standard pattern, similar to the one in Dynawo core. This pattern can be checked by hooks. In order to install this hook, the
```

Comment:
not sure it is the best approach to ask the user to do it. It would be better to propose a script similar to envDynawo.sh that provides the compilation service and handles this automatically

Reply:
DFL has no script for execution and I'm not convinced it should have one. But we can add another issuer for the automation for this step

Reply:
IMHO It would be coherent with all the tools of the dynawo suite and would ease the compilation of DFL and the test execution for newcomers.
Always good to keep coherency between the projects.

Reply:
So we create a new issue for that ?

Reply:
ok!